src/index.tsx

```diff
@@ -436,43 +484,55 @@ export class ScrollBottomSheet<T extends any> extends Component<Props<T>> {
             this.calculateNextSnapPoint(i + 1)
           );
 
-    const runTiming = ({
+    const runAnimation = ({
       clock,
       from,
       to,
       position,
       finished,
+      velocity,
       frameTime,
     }: TimingParams) => {
       const state = {
         finished,
+        velocity: new Value(0),
         position,
         time: new Value(0),
         frameTime,
       };
 
-      const animationParams = {
+      const timingConfig = {
         duration: animationDuration,
-        easing: animationConfig?.easing || DEFAULT_EASING,
+        easing:
+          (props.animationType === 'timing' && props.animationConfig?.easing) ||
+          DEFAULT_EASING,
+        toValue: new Value(0),
       };
 
-      const config = {
+      const springConfig = {
+        ...DEFAULT_SPRING_PARAMS,
+        ...(props.animationType === 'spring' && props.animationConfig),
```

Comment:
That could throw if `animationType === 'spring'` and `animationConfig` is not defined. We need to default it to an empty object.

```suggestion
        ...((props.animationType === 'spring' && props.animationConfig) || {}),
```